pkgs/applications/misc/plasma-theme-switcher/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  stdenv, lib, pkgs, fetchFromGitHub
```

Comment:
we don't pass in `pkgs`, like this. please specify your actual dependencies here. e.g.
```
, cmake
, extra-cmake-modules
, qtbase
, kdeFrameworks
, wrapQtAppsHook
```